tests/README.md

```diff
@@ -1,3 +1,23 @@
-# Integration & Functional Tests for VIC
+VIC Integration & Functional Test Suite
+=======
 
-Integration tests can be run by calling `make integration-tests` from the project's root directory.
+To run the deprecated tests:
+
+1. Integration tests can be run by calling `make integration-tests` from the project's root directory.
+
+To run these tests locally:
+
+1. Create a secrets.yml file that includes:  
+`environment:`  
```

Comment:
Very minor, but I would use triple-backticks above and below the entire block here to make things like this cleaner:

```
environment:  
    ESX_URL: <IP address of your ESX server>  
    ESX_USERNAME: <username you use to login to ESX server>  
    ESX_PASSWORD: <password you use to login to ESX server>  
    ESX_RESOURCE: <resource pool, e.g. /ha-datacenter/host/localhost.localdomain/Resources>
```